b2sdk/exception.py

```diff
@@ -11,10 +11,15 @@
 from abc import ABCMeta
 
 import json
+import re
 import six
 
 from .utils import camelcase_to_underscore
 
+UPLOAD_TOKEN_USED_CONCURRENTLY_ERROR_MESSAGE_RE = re.compile(
+    r'more than one upload using auth token (?P<token>.*)'
```

Comment:
$ and ^ please, and I'm pretty sure `.*` can at least be specified to have no `)` in it and a non-zero length string

Reply:
@ppolewicz fixed in https://github.com/reef-technologies/b2-sdk-python/pull/53/commits/e5337bdd42075ceb2a0f11ee7bbc585b343daf4a